### PR TITLE
Boostrap create-react-app for typescript

### PR DESCRIPTION
**Pull request by bertearazvan**
Created the boostrap for the `create-react-app` using typescript. 
`npx create-react-app my-app --template typescript`

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    },
+    {
+      "src": "logo192.png",
```
Comment (Robert-OP) [inline public/manifest.json]: include dhi logo's for the website up top

File: src/setupTests.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```
Comment (Robert-OP) [inline src/setupTests.ts]: don't think we need this file

File: src/serviceWorker.ts [hunk 3]
```diff
@@ -0,0 +1,149 @@
+// This optional code is used to register a service worker.
```
Comment (Robert-OP) [inline src/serviceWorker.ts]: don't think we need this file since we are not going with a progressive web app (pwa)

**Review by Robert-OP — APPROVED** [review]
small changes, can be solved later on

**Review by JeroenDerks — COMMENTED** [review]
We should include an IE warning similar like we do on: https://github.com/DHI-GRAS/eo-clinic-app/blob/master/public/index.html
ps. there might be many other ways to do this, but this one is working most consistently from my experience

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (JeroenDerks) [inline public/index.html:30]: We should include an IE warning similar like we do on: https://github.com/DHI-GRAS/eo-clinic-app/blob/master/public/index.html
ps. there might be many other ways to do this, but this one is working most consistently from my experience

Comment (bertearazvan) [reply public/index.html:30]: I have followed the same approach

**Review by JeroenDerks — COMMENTED** [review]
Good stuff!

**Review by Robert-OP — COMMENTED** [review]
👍

**Pull request merged**